### PR TITLE
HBW-108 Download Font Awesome when widget is loaded

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 v1.6.1 [unreleased]
 -------------------
 
+### Features
+-   [#246](https://github.com/latera/homs/pull/246) Compile widget with Font Awesome.
```
Comment (TimAle) [inline CHANGELOG.md]: Let's be honest, widget is not _compiled_ with FontAwesome now, widget will download it when page loaded.

File: hbw/app/javascript/packs/hbw/components/form/checkbox.js.jsx [hunk 2]
```diff
@@ -38,11 +38,10 @@ modulejs.define('HBWFormCheckbox', ['React'], (React) => {
       const { tooltip } = this.props.params;
       const { label } = this.props.params;
       const labelCSS = `hbw-checkbox-label ${this.props.params.label_css || ''}`;
-
       return <div className={inputCSS} title={tooltip}>
         <div className="form-group">
           <label className={labelCSS}>
-            <input type='checkbox' {...opts} onChange={this.handleChange} checked={this.state.value} />
+            <input type='checkbox' {...opts} onChange={this.handleChange} checked={this.state.value} className='hbw'/>
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/checkbox.js.jsx]: Use more specific class name.

File: hbw/app/javascript/packs/hbw/components/form/checkbox.js.jsx [hunk 3]
```diff
@@ -19,8 +19,8 @@ modulejs.define('HBWFormCheckbox', ['React'], (React) => {
       };
     },
 
-    handleChange (event) {
-      this.setState({ value: event.target.value });
+    handleChange () {
+      this.setState( prevState => ({ value: !prevState.value }));
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/checkbox.js.jsx]:
```suggestion
      this.setState(prevState => ({ value: !prevState.value }));
```

**Comment by TimAle** [comment]
Some tests are red 🛑

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**